sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -189,12 +189,16 @@ SUBROUTINE MDLFLD
 !     SECOND, STANDARD NGM SEA LEVEL PRESSURE.
       IF (IGET(105) > 0) THEN
          CALL NGMSLP
-!$omp parallel do private(i,j)
+         mygmax=slp(1,jsta)
```

Comment:
@junwang-noaa Are the changes of "mygmax" just for your code debugging or other usages?

Reply:
removed